extensions/linear/src/components/CreateIssueForm.tsx

```diff
@@ -260,26 +275,21 @@ export default function CreateIssueForm(props: CreateIssueFormProps) {
           <Action.SubmitForm onSubmit={handleSubmit} title="Create Issue" />
         </ActionPanel>
       }
-      isLoading={isLoadingTeams}
+      isLoading={isLoadingTeams || isLoadingUsers || props.isLoading}
     >
-      {hasMoreThanOneTeam ? (
```

Comment:
Initially I removed this condition so that the dropdown does not dissapear if a typeahead query yields <= 1 team. But with further tweaking, I am bringing it back in next revision.